### DISCUSSION TITLE
Julia 1.10 becomes LTS version - version support decisions

### DISCUSSION THREAD

**Discussion by SamuelBrand1** [CDCgov/Rt-without-renewal#487] [General]
See here https://julialang.org/blog/2024/10/julia-1.11-highlights/

From our pov we have julia version compat `julia = ">= 1.9"`. 

Would it make sense to change this to be `1.10` and later or possibly `1.10` and "whatever latest version is"?

**Comment by SamuelBrand1** [comment]
As well as `compat` maybe we should change 

https://github.com/CDCgov/Rt-without-renewal/blob/7b6b9c13d499e3134f5f07e43977735abf2a2481/.github/workflows/test-EpiAware.yaml#L20

to `lts` and `1`? I think at the moment thats equivalent to what we have but it'll potentially catch a future shift.

**Comment by seabbs** [comment]
I agree. I think whilst in development we should really just commit to supporting the latest stable version and then make this more complex from here?

**Comment by seabbs** [comment]
Once we are happier with where we are I think it makes sense to talk about versions to support (which I assume is the latest stable and the lts?)

**Comment by SamuelBrand1** [comment]
That makes sense. I don't think this is really an issue with that decision so shifting to discussion.
